### PR TITLE
Support new WordPressAuthenticator configuration and API methods

### PR DESCRIPTION
**Pull request by mindgraffiti**
This task allows WordPress-iOS to support the new configuration and API method changes found in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/10

## To test login changes
1. Clean project
2. `pod install`
3. Run project

**Expected Behavior**
There should be no changes to the Login UI in WordPress-iOS. If there are, then a style definition was incorrectly applied or incorrectly defined in `WordPressAuthenticator-iOS`.

## To test Aztec FancyAlert prompt changes
1.  Revert the `HACK2` commit
2. `pod install`
3. Run project
4. Go to a test blog or throwaway blog account
5. Attempt to write a post
6. A "confirm email" prompt should appear

**Expected Behavior**
The confirm email prompt buttons should remain styled the same as they were before. 

Before changes:

After changes:
after copy

### CODE REVIEW

File: WordPress/Credentials/gencredentials.rb [hunk 1]
```diff
@@ -194,6 +204,8 @@ def print_class(client, secret, pocket, crashlytics, hockeyapp, googleplus, goog
       google_id = value
     elsif k == "GOOGLE_LOGIN_SERVER_ID"
       google_login_server = value
+    elsif k == "GOOGLE_AUTH_SCHEME"
```
Comment (nheagy) [inline WordPress/Credentials/gencredentials.rb]: This should be `GOOGLE_LOGIN_SCHEME_ID`

Comment (mindgraffiti) [reply WordPress/Credentials/gencredentials.rb]: ✔️

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 2]
```diff
@@ -184,9 +184,7 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
         returnValue = YES;
     }
 
-    if ([WordPressAuthenticator isGoogleAuthURLWithUrl:url
-                                     sourceApplication:options[UIApplicationOpenURLOptionsSourceApplicationKey]
-                                            annotation:options[UIApplicationOpenURLOptionsAnnotationKey]]) {
+    if ([WordPressAuthenticator.shared handleGoogleAuthUrl:url sourceApplication:options[UIApplicationOpenURLOptionsSourceApplicationKey] annotation:options[UIApplicationOpenURLOptionsAnnotationKey]]) {
```
[the remark this answers is not in the document]
Comment (mindgraffiti) [reply WordPress/Classes/System/WordPressAppDelegate.m]: ✔️

**Comment by mindgraffiti** [comment]
Tweaked the corner radius to match the same size as the original design. Per p1531239322000162-slack-C6H8C3G23, I'm going with the stronger shadow, offset height 2. 

Tweaked the font styling - I found the original inherited font settings for the titleLabel.

offset2 copy

**Comment by mindgraffiti** [comment]
Hound is stuck. `rake lint` in command line produced no errors. Merging.

**Pull request merged**